wapi/util.py

```diff
@@ -58,15 +58,18 @@ def __init__(self, id=None, name=None, frequency=None, time_zone=None, tag=None,
         self.points = points
         #
         # input_dict is the json dict from WAPI
+        attributes = ['id', 'name', 'frequency', 'time_zone',
+                      'tag', 'issue_date', 'curve_type', 'points']
         if input_dict is not None:
-            for attr in ('id', 'name', 'frequency', 'time_zone', 'tag', 'issue_date', 'curve_type', 'points'):
+            for attr in attributes:
```

Comment:
Dette virker som en endring bare for å endre ting?  Hvorfor liste og ikke tuple?

Reply:
Gitt at det er en liste med ting så virket liste som en mer naturlig datastruktur.

Reply:
Virker rimelig.